save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestSuitesService.kt

```diff
@@ -119,6 +122,21 @@ class TestSuitesService(
         }
     }
 
+    /**
+     * @param organizationName
+     * @return [Flux] of [TestSuite]s by [organizationName]
+     */
+    fun findTestSuitesByOrganizationName(organizationName: String): Flux<TestSuite> = blockingToFlux {
```

Comment:
taking into account that all repositories calls are blockings, I don't think it's useful to return `Flux` or `Mono` as return type in such services: chain calls of methods from such services don't need to be wrapped to separate mono\flux

Reply:
Ok, will make it to be a usual List and will take it into consideration

Reply:
Done